lib/blocs/authentication/viewmodels/authentication_bloc.dart

```diff
@@ -23,27 +24,24 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
       yield state.copyWith(authStatus: AuthStatus.unlocked);
     }
     if (event is OnCreateAccount) {
-      settingsStorage.saveAccount(event.account, event.privateKey);
+      SaveAccountUseCase().run(accountName: event.account, privateKey: event.privateKey);
       // New account --> re-start auth status
       add(const InitAuthStatus());
-      // Set fcm token must be last instruction to allow login, even if there is an error here.
-      await FirebaseMessageTokenRepository().setFirebaseMessageToken(settingsStorage.accountName);
     }
     if (event is OnImportAccount) {
-      settingsStorage.saveAccount(event.account, event.privateKey);
+      SaveAccountUseCase().run(accountName: event.account, privateKey: event.privateKey);
       settingsStorage.privateKeyBackedUp = true;
       // New account --> re-start auth status
       add(const InitAuthStatus());
-      // Set fcm token must be last instruction to allow login, even if there is an error here.
-      await FirebaseMessageTokenRepository().setFirebaseMessageToken(event.account);
     }
     if (event is OnRecoverAccount) {
-      settingsStorage.saveAccount(event.account, event.privateKey);
```

Comment:
I left the code in because
- it is not harmful
- it's very hard to test recovery
- it was in there before
- it's not part of this PR to clean this up